Bottom: eae61c3bf8b09e14ff67ca2242f2794d291aa3c6
Top:    586edbe9472359afd7a1cbf72b3e203f4d650213
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 15:40:56 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c234067..b35ead1 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -501,16 +501,6 @@ object3d_from_board_outline (void)
   return object;
 }
 
-void
-object3d_test_board_outline (void)
-{
-  object3d *board_outline;
-
-  board_outline = object3d_from_board_outline ();
-//  object3d_export_to_step (board_outline, "object3d_test.step");
-  destroy_object3d (board_outline);
-}
-
 object3d *
 object3d_from_tracking (void)
 {
@@ -698,3 +688,13 @@ object3d_from_tracking (void)
 
   return object;
 }
+
+void
+object3d_test_board_outline (void)
+{
+  object3d *board_outline;
+
+  board_outline = object3d_from_board_outline ();
+//  object3d_export_to_step (board_outline, "object3d_test.step");
+  destroy_object3d (board_outline);
+}
